plugins/scaffolder-backend/src/service/router.test.ts

```diff
@@ -1198,11 +1221,22 @@ data: {"id":1,"taskId":"a-random-id","type":"completion","createdAt":"","body":{
               createdBy: '',
             },
           ],
+          totalTasks: 1,
         });
 
         const response = await request(app).get(`/v2/tasks`);
         expect(taskBroker.list).toHaveBeenCalledWith({
           createdBy: undefined,
+          pagination: {
+            limit: undefined,
+            offset: undefined,
+          },
+          filters: {
+            createdBy: undefined,
+            status: undefined,
+          },
+          order: undefined,
+          status: undefined,
```

Comment:
Wouldn't it be the same if to remove the explicit declaration of this params? (i believe all of them are optional)